nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/api/ops/impl/layers/convolution/DepthwiseConv2D.java

```diff
@@ -49,11 +52,14 @@
  */
 @Slf4j
 @Getter
-@NoArgsConstructor
 public class DepthwiseConv2D extends DynamicCustomOp {
 
     protected Conv2DConfig config;
 
+    public DepthwiseConv2D(SameDiff sameDiff, SDVariable input, SDVariable weights, SDVariable bias, Conv2DConfig conv2DConfig) {
+        this(sameDiff, new SDVariable[]{input, weights, bias}, conv2DConfig);
```

Comment:
Bias is optional, need `@NonNull` on input/weights and wrapFilterNull instead of new SDVariable[]{...}

Reply:
Fixed.